Xamarin.Forms.Build.Tasks/XamlCTask.cs

```diff
@@ -44,24 +48,29 @@ public override bool Execute(out IList<Exception> thrownExceptions)
 				return true;
 			}
 
-			var resolver = new XamlCAssemblyResolver();
-			if (!string.IsNullOrEmpty(DependencyPaths))
+			var resolver = DefaultAssemblyResolver ?? new XamlCAssemblyResolver();
+			var xamlCResolver = resolver as XamlCAssemblyResolver;
+
+			if (xamlCResolver != null)
```

Comment:
So if `DefaultAssemblyResolver` is not a valid `XamlCAssemblyResolver`, then we don't do any of this work. Should we at least log a warning?